nix/nixos/cardano-node-service.nix

```diff
@@ -102,14 +100,6 @@ in {
           (the blockchain protocols running cardano).
         '';
       };
-      instanced = mkOption {
-        type = types.bool;
-        default = false;
-        description = ''
-          Whether to enable systemd service instancing.
-          For details see https://fedoramagazine.org/systemd-template-unit-files/
-        '';
-      };
       instances = mkOption {
```

Comment:
Isn't this also coming along with `instanced`, or is this the other instancing implementation?

Reply:
no, `instances`` is not implemented through systemd template, just regular nix-generated services.